src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,38 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int NO_KEY_FOUND = -1;
+    private K[] keys = (K[]) new Object[10];
+    private V[] values = (V[]) new Object[10];
+    private int size = 0;
+
     @Override
     public void put(K key, V value) {
+        if (getIndexOfKey(key) == NO_KEY_FOUND) {
+            keys[size] = key;
+            values[size] = value;
+            size++;
+        } else if (getIndexOfKey(key) >= 0) {
+            values[getIndexOfKey(key)] = value;
```

Comment:
```suggestion
		int keyIndex = getIndexOfKey(key);
        if (keyIndex == NO_KEY_FOUND) {
            keys[size] = key;
            values[size] = value;
            size++;
            return;
        } 
        values[keyIndex] = value;
```

I think Ira wanted to see something like that